USDA_videos/youtube_video_list.py

```
@@ -0,0 +1,62 @@
+import os
+import json
+import googleapiclient.discovery
+
+def get_channel_videos(channel_id):
```

Ruff (0.14.10)

Unused function argument: `channel_id`

(ARG001)